published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,105 @@
+# IPFS Weekly #5
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [February 1](//github.com/ipfs/pm/issues/88) sprint:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+	- [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated
+	- [@lgierth](//github.com/lgierth) automated the [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs 
```

Comment:
[@lgierth](//github.com/lgierth) automated the [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs. The new image `ipfs/go-ipfs` is built automatically for every commit, and has automatic tags for the coming releases. It's also a few MB smaller, and doesn't require mounting a volume for the IPFS repository anymore. Instead, if there's no volume mounted, it'll generate an ephemeral identity and configuration which will be lost when the container exits. This is ideal for testing or just trying something out real quick. Images for go-ipfs 0.3.x are not provided, because the Dockerfile shipped with these version isn't suitable for Docker Hub automatic builds.